dart/lib/src/scope.dart

```diff
@@ -1,5 +1,4 @@
 import 'dart:collection';
-
```

Comment:
nit : the Dart analyzer `organize imports` command add a blank line between `dart:`, `package:` and relative imports